tests/unit_tests/data/test_aggregation.py

```diff
@@ -14,6 +14,7 @@
 # -------------------------------------------------------------------------------------------------
 
 from datetime import timedelta
+from decimal import ROUND_HALF_UP
```

Comment:
I'm interesting to know the thinking behind `ROUND_HALF_UP` vs the standard `ROUND_HALF_EVEN`?

Reply:
I didn't even know ROUND_HALF_EVEN existed, I just asked an LLM for helping me solve the the rounding problem. I guess I used something that worked int his case and was rounding correctly.

Reply:
Related https://gerrytan.wordpress.com/2014/02/25/whats-the-deal-with-half-up-and-half-even-rounding/ although it doesn't matter in this case as it's for a test, you can change to half even if you want in another commit so people can do by analogy in the future.